src/components/editor_flexbox.tsx

```diff
@@ -1,83 +1,133 @@
+import {FC} from 'react'
+
 import Markdown from 'markdown-to-jsx';
-import React, { ChangeEvent, useState } from 'react';
+import React, { ChangeEvent, useEffect, useState } from 'react';
 import StringInput from './string_input';
+import StringInputWrapper from './string_input_wrapper';
 import USStateSelectWrapper from './us_state_select_wrapper';
 import PhotoInput from './photo_input';
+import PhotoInputWrapper from './photo_input_wrapper';
 import DateInputWrapper from './date_input_wrapper';
 import Collapsible from './collapsible';
 import FigureWrapper from './figure_wrapper';
 import NumberInputWrapper from './number_input_wrapper';
 import PrintSection from './print_section';
 import SelectWrapper from './select_wrapper';
 import TextInput from './text_input';
+import MdxTemplateView from '../components/mdx_template_view'
+import { StoreContext, StoreProvider } from './store';
+import MdxWrapper from './mdx_wrapper';
+import Tab from 'react-bootstrap/Tab'
+import Tabs from 'react-bootstrap/Tabs'
+import {runSync, run} from '@mdx-js/mdx'
+import {evaluateSync,EvaluateOptions,compileSync} from '@mdx-js/mdx'
+// import {Fragment as _Fragment, jsx as _jsx} from 'react/jsx-runtime'
+import * as runtime from 'react/jsx-runtime' // Development.
+import {Fragment as _Fragment, jsx as _jsx, jsxs as _jsxs} from 'react/jsx-runtime'
+import * as provider from '@mdx-js/react'
+import { useMDXComponents } from '@mdx-js/react';
+import PhotoWrapper from './photo_wrapper';
+import TableWrapper from './table_wrapper';
+import TextInputWrapper from './text_input_wrapper';
+import DateStr from './date';
 
-//const md = `<StringInput label="Street Address" path="location.street_address" /><StringInput label="City" path="location.city" /><USStateSelect label="State" path="location.state" />`
-
-
-
-const EditorFlexBox = () => {
-
-  const [text, setText] = useState('');
+const components = {
+  Collapsible,
+  DateInput: DateInputWrapper,
+  Figure: FigureWrapper,
+  NumberInput: NumberInputWrapper,
+  Photo: PhotoWrapper,
+  PhotoInput: PhotoInputWrapper,
+  PrintSection,
+  Select: SelectWrapper,
+  StringInput: StringInputWrapper,
+  Table: TableWrapper,
+  TextInput: TextInputWrapper,
+  USStateSelect: USStateSelectWrapper,
+  DateStr: DateStr,
+  Tab: Tab,
+  Tabs: Tabs,
+};  
 
-  const handleChange = (event: ChangeEvent<HTMLTextAreaElement>) => {
-    setText(event.target.value);
-    event.target.style.height = 'auto'; // Reset the height to auto to adjust the size
-    event.target.style.height = `${event.target.scrollHeight}px`; // Set the height to the scrollHeight to adjust the size
-  };
 
+const EditorFlexBox: FC = () => {
+  const initialState = `
```

Comment:
I suggest calling this something more description, such as `initialTextAreaContent`.